### PR TITLE
App design

### PR DESCRIPTION
**Pull request by AlexDresco**
Need to finish the Input page and do the results page

### CODE REVIEW

File: components/InputManual.js [hunk 1]
```diff
@@ -14,23 +14,46 @@ class InputManual extends React.Component {
...
-         <Button
-          title="Submit number!"
+         <TouchableHighlight
+          // style={styles.buttonText}
           onPress = { () => {
             this.props.navigateToResult();
-          }}
-          />
+          }}>
+          <Text>Submit barcode</Text>
```
Comment (Ruaridhi) [inline components/InputManual.js]: text needs to be move centrally / repositioned generally(would be my preference)

File: components/InputScanner.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ class InputScanner extends React.Component {
           <Text>Camera permission is not granted</Text> :
           <BarCodeScanner
               onBarCodeRead={this._handleBarCodeRead}
-              style={{ height: 200, width: 200 }}
+              style={{ height: '100%', width: 500 }}
```
Comment (Ruaridhi) [inline components/InputScanner.js]: is there not a way to make the width reactive to the screen size, rather then fixed which may cause style differences across devices

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -1,37 +1,85 @@
...
+    fontFamily: 'argana',
+    fontSize: 65,
+    color: 'gray',
+  },
+  buttonContainer: {
+    borderRadius: 10,
+    width: 250,
+    height: 55,
+    marginBottom: 150,
+    backgroundColor: "#E4DFC1"
```
Comment (Ruaridhi) [inline screens/HomeScreen.js]: still would like to play with the opacity(but thats very minor)

File: screens/InputScreen.js [hunk 4]
```diff
@@ -6,6 +6,10 @@ import InputScanner from '../components/InputScanner';
 import InputManual from '../components/InputManual';
 
 class InputScreen extends React.Component {
+  static navigationOptions = {
+    header: null,
```
Comment (Ruaridhi) [inline screens/InputScreen.js]: is this what gets rid of the white space at the top? and how if so?

File: components/InputManual.js [hunk 5]
```diff
@@ -14,23 +14,46 @@ class InputManual extends React.Component {
...
+          </TouchableHighlight>
```
[the remark this answers is not in the document]
Comment (AlexDresco) [reply components/InputManual.js]: @Ruaridhi It's basically a button, Button doesn't take a style props but Touchable is.

File: screens/InputScreen.js (hunk 4 above)
Comment (AlexDresco) [reply screens/InputScreen.js]: Yes :) It changes the page header using React navigation

**Pull request merged**